site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -25,6 +25,27 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
     - Initiate via node CLI to selectively run specs based on test metadata, e.g. `node run_tests.js --group='@accessibility'` which will run all specs with `@accessibility` metadata.
 3.  Tests are executed according to your selection and will display whether the tests passed or failed.
 
+### Environment Variables
+There are several environment variables we're using for Cypress testing with the primary purpose:
```

Comment:
This just needs a line break between the header and content.

Reply:
Done. Also, change similar format in this page.